.github/workflows/publish-to-github-pages.yml

```diff
@@ -9,11 +9,15 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      ALGOLIA_APP_ID: ${{ secrets.ALGOLIA_APP_ID }}
+      ALGOLIA_SEARCH_KEY: ${{ secrets.ALGOLIA_SEARCH_KEY }}
+      ALGOLIA_ADMIN_KEY: ${{ secrets.ALGOLIA_ADMIN_KEY }}
     steps:
       - uses: actions/checkout@master
 
       - uses: enriikke/gatsby-gh-pages-action@v2
         with:
           # access-token is a GitHub Personal access token with the "repo" scope
           access-token: ${{ secrets.PUBLIC_REPO_ACCESS_TOKEN }}
-          deploy-branch: gh-pages
+          deploy-branch: gh-pages
```

Comment:
Very minor, we prefer to have empty line at the end of file.

Reply:
Sorry